demo/opae-nlb-demo/Dockerfile

```diff
@@ -13,24 +13,21 @@ FROM ${CLEAR_LINUX_BASE} as builder
 ARG CLEAR_LINUX_VERSION=
 
 RUN swupd update --no-boot-update ${CLEAR_LINUX_VERSION} && \
-    swupd bundle-add wget c-basic devpkg-json-c devpkg-util-linux devpkg-hwloc doxygen patch
+    swupd bundle-add wget c-basic devpkg-json-c devpkg-util-linux devpkg-hwloc devpkg-tbb git
```

Comment:
What is the reason to install devpkg-tbb ? Isn't it way bigger than actually needed "doxygen"?

Reply:
It's a build dependency. Build fails without it with this error:
```
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
TBB_LIBRARIES
```

Why do we need doxygen if all what we need is nlb0 and nlb3 binaries and their runtime dependencies?
How doxygen is related to having or not having libtbb build dependency?

Reply:
Doxygen was previously a build dependency even for nlb0/3. If we don't need it anymore, great.
If tbb is new build dependency, also fine.